web/src/components/layout/user-menu.tsx

```diff
@@ -38,23 +40,31 @@ export default function UserMenu() {
         className={'user-menu ' + (showMenu ? 'active' : '')}
         onMouseEnter={() => setShowMenu(true)}
         onMouseLeave={() => setShowMenu(false)}>
+        <audio preload="auto" ref={audioRef} />
         <button
           className="toggle"
           title="click to play avatar"
-          onClick={() => {
```

Comment:
For mobile we'd still need tap to listen, so I think we'd wanna have both mouseEnter and click